pkg/watch/term_unix_other.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2021 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
Is it OK to have this when the odo code is licensed as Apache Licence 2.0?

Reply:
It is the license for the Go library and the code coming from `golang.org/x/*` repos.
I can imagine it is ok

This code specifically comes from the `golang.org/x/term` repository (https://cs.opensource.google/go/x/term/+/a9ba230a:term_unix_other.go)